README.rst

```diff
@@ -1,6 +1,8 @@
 ################
 The Lago Project
 ################
+** Lago project is EOL. This repository is no longer available for pushes. **
+
 Lago is an add-hoc virtual framework which helps you build virtualized
```

Comment:
Let's get rid of the description and especially the link to the docs below - they are outdated anyway.